### PR TITLE
add new features for the JSON

### PR DESCRIPTION
**Pull request by calmacx**
* allow a route in the API to download the JSON
* switch so the name of the concept mapping (e.g. FEMALE) is use in the json name

### CODE REVIEW

**Review by shahzadmumtaz22 — APPROVED** [review]

**Comment by spco** [comment]
@calmacx could you just add an item to the changelog particularly as this is a compatibility change. Thanks

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Please append a line to the changelog for each change made.
 
+## v1.2
+* Mapping rules json structural change, now objects associated to a CDM table are a dictionary (with a key name) rather than just a list.
+
```
Comment (calmacx) [inline changelog.md:7]: @spco is this fine?

Comment (spco) [reply changelog.md:7]: It needs appending to v1.1-beta list please

Comment (calmacx) [reply changelog.md]: that makes a lot more sense - slow morning -  thanks, done

**Pull request merged**